### PR TITLE
Feature/simply 3517 text updates

### PR DESCRIPTION
**Pull request by kaplanmichael**
https://jira.nypl.org/browse/SIMPLY-3517

### CODE REVIEW

**Comment by kristojorg** [comment]
Looks like there is a minor lint error on this. Otherwise looks good, I'm just waiting for access to the copy document to finish review

**Comment by kaplanmichael** [comment]
> Looks like there is a minor lint error on this. Otherwise looks good, I'm just waiting for access to the copy document to finish review

Fixed!

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@
 - Fix: Protect against invalid redirect uris caused by mismatch in hydration.
 - Fix: Improve the appearance of home page book thumbnails for larger viewports
 - tests: Add a Github Workflow to trigger integration tests in NYPL-Simplified/integration-tests-web.
+- Update: Update landing page copy.
```
Comment (kristojorg) [inline CHANGELOG.md:14]: This should be moved to unreleased changes, this probably happened during the merge from dev -> this branch

**Review by kristojorg — CHANGES_REQUESTED** [review]
Can you just move the changelog and then we can merge this?

File: CHANGELOG.md (hunk 1 above)
Comment (kaplanmichael) [reply CHANGELOG.md]: Done!

**Review by kristojorg — APPROVED** [review]
Looks good

**Pull request merged**